frontend/lib/src/components/shared/Toolbar/Toolbar.tsx

```diff
@@ -40,17 +40,15 @@ export interface ToolbarActionProps {
   onClick: () => void
 }
 
-export function ToolbarAction({
-  label,
-  show_label,
-  icon,
-  onClick,
-}: ToolbarActionProps): ReactElement {
+export function ToolbarAction(props: ToolbarActionProps): ReactElement {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
nit: instead of silencing the warning, would it make sense to rename the props (likely `show_label`, right?) or is this beyond our control?
If we cannot rename the incoming props, would it remove the warning if we renamed it during restructuring like `const { label, show_label: showLabel, icon, onClick } = props`?

Reply:
i think i can actually revert this change completely since i no longer need the test id here